source/reference/method/MongoDBCollection-listSearchIndexes.txt

```diff
@@ -40,16 +40,16 @@ Parameters
 
      * - batchSize
        - integer
-       - Specifies the batch size for the cursor, which will apply to both the
-         initial ``aggregate`` command and any subsequent ``getMore`` commands.
-         This determines the maximum number of documents to return in each
-         response from the server.
-
-         A batchSize of ``0`` is special in that and will only apply to the
-         initial ``aggregate`` command; subsequent ``getMore`` commands will use
-         the server's default batch size. This may be useful for quickly
-         returning a cursor or failure from ``aggregate`` without doing
-         significant server-side work.
+       - The maximum number of documents within each batch returned in a query result, which applies 
+         to the ``aggregate`` command. By default, the ``aggregate`` command has an initial batch size of 
+         ``101`` documents and a maximum size of 16 mebibytes for each subsequent batch. This
+         option can enforce a smaller limit than 16 mebibytes, but not a larger
+         one. If you set ``batchSize`` to a limit that results in batches larger than
+         16 MiB, this option has no effect.
+
+         A batchSize of ``0`` means that the cursor will be established, but no documents 
+         will be returned in the first batch. This may be useful for quickly returning a cursor
+         or failure from ``aggregate`` without doing significant server-side work.
```

Comment:
I don't think we need to discuss "batchSize of 0" for the `listSearchIndexes()` helper. Consider removing this paragraph.

It makes more sense for general `aggregate()` helpers and to a lesser extent `find()`. But the fact that listing search indexes uses `aggregate` is really more of an implementation detail that users won't care about.